web/controllers/api/docs_controller.ex

```diff
@@ -28,6 +28,7 @@ defmodule HexWeb.API.DocsController do
 
   def delete(conn, _params) do
     revert(conn.assigns.release)
+    publish_sitemap()
```

Comment:
This function should run inside the async task, i.e. inside the `revert/1` function.